packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -84,7 +84,7 @@ export interface IColumnHeaderCellProps extends IColumnNameProps, IProps {
     menuIconName?: string;
 
     /**
-     * CSS styles for the top level element
+     * CSS styles for the top level element.
```

Comment:
note that trailing periods are going to be hard to enforce. you should write a TSLint rule for it.